ansys/mapdl/core/mapdl.py

```diff
@@ -2246,13 +2246,13 @@ def run(self, command, write_to_log=True, mute=None, **kwargs):
 
         if command[:3].upper() in INVAL_COMMANDS:
             exception = RuntimeError(
-                'Invalid pymapdl command "%s"\n\n%s'
+                'Invalid PyMAPDL command "%s"\n\n%s'
                 % (command, INVAL_COMMANDS[command[:3].upper()])
             )
             raise exception
         elif command[:4].upper() in INVAL_COMMANDS:
             exception = RuntimeError(
-                'Invalid pymapdl command "%s"\n\n%s'
+                'Invalid PyMAPDL command "%s"\n\n%s'
                 % (command, INVAL_COMMANDS[command[:4].upper()])
```

Comment:
Whenever we fix something like this, let's converge on f-strings. I don't think doing a big refactor makes sense now, but since we're going through the trouble of doing this, we can work on improving our code base.

Reply:
But not this PR, next one. This is good for now.

Reply:
Makes perfect sense.